DuckDuckGo/BrokenSiteCategoryCell.swift

```diff
@@ -0,0 +1,26 @@
+//
```

Comment:
Sorry I didn't spot this earlier, but this seems redundant.  Can we just use a basic cell since it's just a label?

Reply:
Sure, my bad! :)